test/e2e/wireguard_test.go

```diff
@@ -45,16 +45,20 @@ func TestWireGuard(t *testing.T) {
 		skipIfMissingKernelModule(t, data, node.name, []string{"wireguard"})
 	}
 	var previousTrafficEncryptionMode string
+	var previousMulticastEnabledState bool
```

Comment:
ditto: changes to this file can be reverted?

Reply:
removed

Reply:
We may still need this change, otherwise, "E2e tests on a Kind cluster on Linux with all features enabled" may fail as Multicast was enabled before.

Reply:
Makes sense. It would have been nice to have a comment here explaining that enabling WireGuard requires disabling Multicast, but no big deal, I see that CI jobs have passed already